docs/api/ref/DesktopAgent.md

```diff
@@ -20,6 +20,7 @@ interface DesktopAgent {
   findIntent(intent: string, context?: Context): Promise<AppIntent>;
   findIntentsByContext(context: Context): Promise<Array<AppIntent>>;
   raiseIntent(intent: string, context: Context, target?: string): Promise<IntentResolution>;
+  raiseContext(context: Context): Promise<IntentResolution>;
```

Comment:
I would prefer if we can call this `raiseIntentForContext`. 

There are two reasons for this:
1. You can't raise a "context", that doesn't make sense intuitively.
2. `raiseIntent` and `raiseIntentForContext` would follow the same pattern as `findIntent` and `findIntentsByContext`, one of which takes an intent name, and the other just a context. It's good symmetry.

Reply:
That's a good suggestion. Also makes the explaining it simpler. I'll update the PR and docs.

Reply:
maybe 'raiseIntentsByContext' ?